frontend/lib/l10n/app_en.arb

```diff
@@ -1,133 +1,129 @@
-  "identification_scanningFailed": "Fehler beim Lesen des Codes",
-  "identification_selfieCamera": "Frontkamera",
-  "identification_standardCamera": "Standard-Kamera",
-  "identification_stopShowing": "Nicht mehr anzeigen",
-  "identification_timeIncorrect": "Die Uhrzeit Ihres Geräts scheint nicht zu stimmen. Bitte synchronisieren Sie die Uhrzeit in den Systemeinstellungen.",
-  "identification_title": "Ausweisen",
-  "identification_unlimited": "unbegrenzt",
-  "identification_validFromUntil": "Gültig: {startDate} bis {expirationDate}",
-  "identification_validUntil": "Gültig bis: {expirationDate}",
+  "identification_activate": "Activate",
+  "identification_activateCardCurrentDevice": "Activate card on this device?",
+  "identification_activateCardCurrentDeviceRationale": "Your card is already activated on another device. If you activate your card on this device, it will be automatically deactivated on your other device.",
+  "identification_activationInvalidState": "The activation is in an invalid state.",
+  "identification_authenticationPossible": "You can use this QR code to identify yourself at acceptance points:",
+  "identification_cameraAccessRequired": "Access to camera required",
+  "identification_cameraAccessRequiredSettings": "To scan a QR code, the app needs access to the camera.\nYou can allow the app to access the camera in the settings.",
+  "identification_cardAlreadyActivated": "The scanned QR code has already been activated.",
+  "identification_cardExpired": "Your card has expired.\nUnder \"More actions\" you can apply for renewal.",
+  "identification_cardInvalid": "Your card is invalid.\nIt has either been revoked or activated on another device.",
+  "identification_cardNotYetValid": "The validity period of your card has not started yet.",
+  "identification_checkFailed": "Your card could not be verified. Please make sure you have an internet connection and try again.",
+  "identification_checkRequired": "Verification necessary",
+  "identification_checkingCode": "The QR code is verified by a server request.",
+  "identification_codeExpired": "The scanned code has already expired on {expirationDate}.",
+  "identification_codeInvalid": "The content of the scanned code cannot be understood. It is probably a QR code that was not generated for this app.",
+  "identification_codeInvalidMissing": "The content of the scanned code is incomplete. (Error code: {missing}Missing)",
+  "identification_codeSavingFailed": "The scanned code cannot be saved in the app.",
+  "identification_codeUnknownError": "An unknown error occurred while scanning the QR code.",
+  "identification_codeVerificationFailed": "The scanned code could not be verified by the server.",
+  "identification_codeVerificationFailedConnection": "The scanned code could not be verified. Please make sure you have an internet connection and try again.",
+  "identification_compareWithID": "Verify the displayed data against an official photo ID.",
```

Comment:
Match the displayed data with an official photo ID.
Aber das jetzige geht auch :)